src/traffic participants/EntitySpawner.ts

```diff
@@ -0,0 +1,20 @@
+class EntitySpawner {
+    spawn(id: number, dir: Direction): RoadUser {
```

Comment:
When you use enums, classes, or something else located in another file, you have to export your structure and import it in another place.